src/applications/pact-act/containers/questions/TernaryRadios.jsx

```diff
@@ -74,36 +64,81 @@ const TernaryRadios = ({
 
   return (
     <>
-      <VaRadio
-        data-testid={testId}
-        onBlur={onBlurInput}
-        className="vads-u-margin-bottom--3"
-        error={(formError && 'Select a response.') || null}
-        hint=""
-        label={h1}
-        label-header-level="1"
-        onVaValueChange={onValueChange}
+      <div
+        className={
+          formError
+            ? 'vads-u-margin-bottom--3 form-question-error'
+            : 'vads-u-margin-bottom--3'
+        }
       >
-        {locationList}
-        <VaRadioOption
-          checked={formValue === responses[0]}
-          label={responses[0]}
-          name={shortName}
-          value={responses[0]}
-        />
-        <VaRadioOption
-          checked={formValue === responses[1]}
-          label={responses[1]}
-          name={shortName}
-          value={responses[1]}
-        />
-        <VaRadioOption
-          checked={formValue === responses[2]}
-          label={responses[2]}
-          name={shortName}
-          value={responses[2]}
-        />
-      </VaRadio>
+        <h1 className="form-question-header" id="form-question">
+          {h1}
+        </h1>
+        {locationList ? <div id="form-instructions">{locationList}</div> : null}
+        <fieldset
+          aria-labelledby="form-question form-instructions"
+          data-testid={testId}
+        >
+          {formError && (
+            <span className="usa-error-message" role="alert">
+              <div className="form-text-error">
+                <span className="usa-sr-only">Error</span> Select a response.
+              </div>
+            </span>
+          )}
+          <>
```

Comment:
Curious - do we need these fragments throughout?

Reply:
Moving to a helper function and map we will need these.